Automattic/jetpack
# AAG upgrade banner: do not display on WoA sites

#### Changes proposed in this Pull Request:

* Since WoA sites include Security features, let's not show them the message.

![image](https://user-images.githubusercontent.com/426388/148248042-e873a427-2186-4258-9c25-535fd89a3044.png)

#### Jetpack product discussion

﻿See p8oabR-LV-p2#comment-5732

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a WoA site where you've installed the Beta plugin, switch to this branch.
* Go to Jetpack > Dashboard.
* You should not see the large banner pictured above.
